web/docs/auth/email.md

```diff
@@ -595,7 +584,234 @@ Users can enter their new password there.
 
 The content of the e-mail can be customized, read more about it [here](#passwordreset-passwordresetconfig-).
 
-## Using The Auth
+## Creating a Custom Sign-up Action
+
+> TODO: test out the code below and make sure it works
```

Comment:
I'm not sure if we should encourage custom signup actions for the Email auth since it's really complex and they have to take care of:
- checking for existing users
- the timing attack prevention (faking the work)
- sending the verification email
... 

Maybe the better workflow could be if we provided `preSignup` and `postSignup` hooks. But this is obviously out of scope here. We already talked about such hooks and how they could be useful.

Reply:
I've created an issue for this piece, I'll work on it in the near future: https://github.com/wasp-lang/wasp/issues/1631